js/react_native/android/src/androidTest/java/ai/onnxruntime/reactnative/TensorHelperTest.java

```diff
@@ -207,7 +206,6 @@ public void createInputTensor_double() throws Exception {
   }
 
   @Test
-  @Ignore("data type for Slice is not supported in mobile package")
```

Comment:
thanks for enabling these tests again.
FYI, there are also some similarly disabled iOS tests that we can probably re-enable:

https://github.com/microsoft/onnxruntime/blob/a5d70d8609e6c35a2b75007cdd422beb09752e29/js/react_native/ios/OnnxruntimeModuleTest/TensorHelperTest.mm#L216

https://github.com/microsoft/onnxruntime/blob/a5d70d8609e6c35a2b75007cdd422beb09752e29/js/react_native/ios/OnnxruntimeModuleTest/TensorHelperTest.mm#L224